service.php

```diff
@@ -301,7 +301,13 @@ function page_optimize_get_path( $uri ) {
 	}
 
 	if ( defined( 'PAGE_OPTIMIZE_CONCAT_BASE_DIR' ) ) {
-		$path = realpath( PAGE_OPTIMIZE_CONCAT_BASE_DIR . "/$uri" );
+		if ( file_exists( PAGE_OPTIMIZE_CONCAT_BASE_DIR . "/$uri" ) ) {
```

Comment:
Need to check both paths.

If you need reproducing, refresh `wp-admin` in the plugin editor, and you'll see it's trying to load the `spinner` from `__wp__/wp-includes`.